rtv/page.py

```diff
@@ -372,8 +382,11 @@ def _remove_cursor(self):
     def _move_cursor(self, direction):
 
         self._remove_cursor()
-
-        valid, redraw = self.nav.move(direction, len(self._subwindows))
+        if abs(direction) > 1:
+            direction = (len(self._subwindows)-1)*direction/abs(direction)
```

Comment:
Doing it this way will only work if you expect `len(_subwindows)` to always be the same. For example, say your cursor is at the bottom of the screen and you page down. If the next submission in the list is very large and takes up most of the screen, you will scroll right past it. 

What we really need to do is make it so that paging down takes the next submission on the list (the one after the  bottom submission on the screen) and makes it the new top submission on the screen. And likewise for paging up. Does that make sense? I think this will be a bit more complicated to pull off.